app/models/shipit/deploy_spec/npm_discovery.rb

```diff
@@ -79,7 +92,67 @@ def package_version
         JSON.parse(package_json.read)['version']
       end
 
+      def publish_config
+        JSON.parse(package_json.read)['publishConfig']
+      end
+
+      def publish_config_access
+        publish_config['access'] || PRIVATE
+      end
+
+      def package_name
+        JSON.parse(package_json.read)['name']
+      end
+
+      def scoped_package?
+        package_name.start_with?('@shopify')
+      end
+
+      def publish?
+        return false if publish_config.blank?
+        return false unless valid_publish_config_access?
+        return false unless package_scoped_when_private?
+      end
+
+      def valid_publish_config_access?
+        return false unless VALID_ACCESS.include?(publish_config_access)
```

Comment:
I think this is a prolix equivalent of `VALID_ACCESS.include?(publish_config_access)`.

Reply:
Yes, yes it is.
Looks like I was a bit high off some ruby here...🤣